go.mod

```diff
@@ -19,6 +19,7 @@ require (
 	github.com/google/uuid v1.1.1
 	github.com/grpc-ecosystem/go-grpc-middleware v1.2.0
 	github.com/grpc-ecosystem/grpc-gateway v1.9.6
+	github.com/hashicorp/golang-lru v0.5.1
```

Comment:
heads up this library uses MPL 2 license

Reply:
Is this an issue? looking at the FAQ it seems like this doesn't violate the license:
https://www.mozilla.org/en-US/MPL/2.0/FAQ/

From Q8 on the FAQ:
> You may distribute any executables you create under a license of your choosing, as long as that license does not interfere with the recipients' rights to the source under the terms of the MPL.

I guess we are doing some form of "distribution" here. But, the source is here for all to see, and we haven't modified the source of the original lru package with the MP license.

I did see another LRU package with an MIT license if folks are concerned at all though.